### PR TITLE
Add notes to XRPermissionStatus, remove XRPermissionDescriptor

### PR DESCRIPTION
**Pull request by saschanaz**
#### Summary

Split from #10581

### CODE REVIEW

File: api/XRPermissionStatus.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
         "spec_url": "https://immersive-web.github.io/webxr/#xrpermissionstatus",
         "support": {
           "chrome": {
-            "version_added": false
+            "version_added": false,
+            "notes": "See <a href='https://crbug.com/b/1060163'>bug 1060163</a>."
```
Comment (Elchi3) [inline api/XRPermissionStatus.json:10]: Does this 404 just for me?
https://crbug.com/1060163 seems to work.

**Review by Elchi3 — CHANGES_REQUESTED** [review]
Looks good to me but please let me know if there is indeed a 404 or not.

File: api/XRPermissionStatus.json (hunk 1 above)
Comment (saschanaz) [reply api/XRPermissionStatus.json:10]:
```suggestion
            "notes": "See <a href='https://crbug.com/1060163'>bug 1060163</a>."
```

👀

**Review by Elchi3 — APPROVED** [review]
Thanks! 👍

**Pull request merged**